src/test/java/org/opentripplanner/netex/mapping/FlexStopsMapperTest.java

```diff
@@ -82,23 +84,53 @@ public class FlexStopsMapperTest {
     )
   );
 
+  static final Collection<Double> INVALID_AREA_POS_LIST = new ArrayList<>(
+    Arrays.asList(
+      59.62575084033623,
+      6.3023991052849,
+      59.62883380609349,
+      6.289718020117876,
+      59.6346950024935,
+      6.293494451572027
+    )
+  );
+
```

Comment:
```suggestion
  static final Collection<Double> INVALID_AREA_POS_LIST = List.of(
      59.62575084033623,
      6.3023991052849,
      59.62883380609349,
      6.289718020117876,
      59.6346950024935,
      6.293494451572027
    )
  );

```

Reply:
true, and the original test data AREA_POS_LIST could also be simplified.
Done for both